### PR TITLE
Admin edit product

### PR DESCRIPTION
**Pull request by jacquelyn-bradshaw**

### CODE REVIEW

File: controllers/adminController.js [hunk 1]
```diff
@@ -6,12 +6,12 @@ function viewAdmin(req, res) {
   if (!res.locals.isAdmin) {
     return res.status(403).render("403")
   }
-  res.render("admin")
+  res.render("admin/admin")
```
Comment (johnbradshw) [inline controllers/adminController.js]: When splitting out into directories, it's standard practice to name the default view "index.ejs"

File: controllers/productController.js [hunk 2]
```diff
@@ -1,31 +1,21 @@
...
-
-  try {
-    productId = new ObjectId(productId)
-  } catch {
-    return res.status(404).render("404")
-  }
-
-  let product = new Product()
-  product = await product.getProduct(productId)
+  let product = new Product(null, null, null, null, req.params.id)
```
Comment (johnbradshw) [inline controllers/productController.js:11]: Lets talk about the product model

File: models/product.js [hunk 3]
```diff
@@ -18,11 +24,11 @@ class Product {
     return products
   }
 
-  async getProduct(id) {
+  async getProduct() {
```
Comment (johnbradshw) [inline models/product.js:27]: Why did you change this?

**Review by johnbradshw — APPROVED** [review]

File: controllers/adminController.js (hunk 1 above)
Comment (jacquelyn-bradshaw) [reply controllers/adminController.js]: File name changed to index

File: controllers/productController.js (hunk 2 above)
Comment (jacquelyn-bradshaw) [reply controllers/productController.js:11]: Discussed

File: models/product.js (hunk 3 above)
Comment (jacquelyn-bradshaw) [reply models/product.js:27]: Discussed
